components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -80,23 +85,13 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     this.iamMajorVersion$ = store.select(iamMajorVersion);
     this.projectsEnabled$ = store.select(atLeastV2p1);
 
-    this.applyRulesButtonText$ = this.projects.applyRulesStatus$.pipe(
-      map(({ state, percentageComplete }: ApplyRulesStatus) => {
-        switch (state) {
-          case ApplyRulesStatusState.NotRunning:
-            return 'Update Projects';
-          case ApplyRulesStatusState.Running:
-            return `Updating Projects ${Math.round(percentageComplete * 100)}%...`;
-        }
-      })
-    );
-
     this.projects.applyRulesStatus$
-      .subscribe(({ state, failed, cancelled }: ApplyRulesStatus) => {
+      .subscribe(({ state, failed, cancelled, percentageComplete }: ApplyRulesStatus) => {
         if (state === ApplyRulesStatusState.NotRunning) {
           this.applyRulesInProgress = false;
           this.updateProjectsFailed = failed;
```

Comment:
I just noticed there's a bug that I created here. L92-96 should be after the condition block on L90 closes. Otherwise your percentage won't change while the update is running. 😁 
Because the rebase-with-master adds changes, too, I'm just going to paste in what I think the net result should be--see if it makes sense. This includes my next comment on your L94, too:
```
        if (state === ApplyRulesStatusState.NotRunning) {
          if (this.applyRulesInProgress) {
            this.cancelRulesInProgress = false;
            this.closeConfirmApplyStopModal();
          }
          this.applyRulesInProgress = false;
        }
        this.updateProjectsFailed = failed;
        this.updateProjectsCancelled = cancelled;
        if (!this.cancelRulesInProgress) {
          this.percentageComplete = percentageComplete;
        }
```